src/main/java/com/atlassian/jira/cloud/jenkins/util/WebhookUrlValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.jira.cloud.jenkins.util;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+public final class WebhookUrlValidator {
+
+    public static boolean isValid(final String webhookUrl) {
+        try {
+            new URL(webhookUrl).toURI();
+            return true;
```

Comment:
Is it make sense to check for the presence of jenkins_server_uuid parameter?

Reply:
yeah, I thought about that, but decided against it because it would create a dependency between the Jenkins plugin and the Forge app that might prove a hindrance in future development. Having said that, it's a weak reason, so we may end up adding that validation rule at some point.